src/controllers/controlpickermenu.cpp

```diff
@@ -175,6 +175,8 @@ ControlPickerMenu::ControlPickerMenu(QWidget* pParent)
                    tr("Cue button (CDJ mode)"), cueMenu);
     addDeckControl("play_stutter", tr("Stutter Cue"),
                    tr("Stutter cue"), cueMenu);
+    addDeckControl("cue_play", tr("Cue (CUP Mode)"),
+                       tr("Cue button (CUP mode)"), cueMenu);
```

Comment:
IMHO this  the CUP button is a button of its own, not a cue button in a cue mode. 
How about:
```
addDeckControl("cue_play", tr("CUP (Cue Play)),
                          tr("Goto cue point and play after release"), cueMenu);
```